generators/java/sdk/VERSION

```diff
@@ -1 +1 @@
-0.8.11
\ No newline at end of file
+0.9.0-rc0
```

Comment:
wasn't sure on these versions -- pr i based this off of bumped minor version + made it an rc so i imitated